support/docker/production/docker-compose.yml

```diff
@@ -3,24 +3,20 @@ version: "3.3"
 services:
 
   reverse-proxy:
-    image: traefik:v1.7
-    network_mode: "host"
-    command:
-      - "--docker" # Tells Træfik to listen to docker
-      - "--acme.email=${TRAEFIK_ACME_EMAIL}" # Let's Encrypt ACME email
-      - "--acme.domains=${TRAEFIK_ACME_DOMAINS}" # Let's Encrypt ACME domain list
+    image: staticfloat/nginx-certbot@sha256:7c2d5ae62e2737605bc8936f4e9bddefc2b91102d044408be901b4985c86ac9d
+    env_file:
+      - .env
     ports:
       - "80:80"     # The HTTP port
       - "443:443"   # The HTTPS port
     volumes:
-      - /var/run/docker.sock:/var/run/docker.sock # So that Traefik can listen to the Docker events
-      - ./docker-volume/traefik/acme.json:/etc/acme.json
-      - ./docker-volume/traefik/traefik.toml:/traefik.toml
+      - type: bind
```

Comment:
Hello,
This means we need to get the nginx configuration directly in the peertube repo, where before we only get the docker-compose part (with the traefik conf inside). Perhaps you should mention it in the documentation?

Reply:
Yes, that's why it is in TODO.

Reply:
Okay great, I was unsure if it covers new deployment or update existing :+1: